.pre-commit-config.yaml

```diff
@@ -41,18 +43,23 @@ repos:
               README.rst|
               pyproject.toml|
           )$
-  - repo: https://github.com/PyCQA/prospector
-    rev: v1.16.0
+  - repo: local
     hooks:
       - id: prospector
+        name: prospector
+        description: Analyze Python code using Prospector
+        entry: prospector
+        language: python
+        types: [python]
+        require_serial: true
+        verbose: true
         additional_dependencies:
-          - "prospector[with_everything]"
           # the following are needed to run mypy successfully in the pre-commit virtualenv
           - types-setuptools
           - types-PyYAML
         args:
-          - --summary-only
           - --zero-exit
```

Comment:
By the way, why do we use this argument?

Reply:
My guess is that the output of prospector messed with the output of pre-commit without it. Either causes formatting issues or drowns out other hook problems?  I don't remember, maybe I didn't add it. I have no preference either way.

Reply:
Ok, I will try to remove it in another pull request :-)